src/app/public/modules/toast/toast.service.ts

```diff
@@ -85,9 +85,10 @@ export class SkyToastService implements OnDestroy {
 
   /**
    * Opens a new toast using a custom component.
-   * @param component Angular component to inject into the toast body
-   * @param config Optional configuration
-   * @param providers Optional providers for the custom component
+   * @param component Specifies an Angular component to inject into the toast body,
+   * @param config Specifies additional configuration options for the toast.
+   * @param providers Specifies an array of custom providers to pass to the custom component's
+   * constructor. Returns a `SkyToastInstance`.
```

Comment:
Should "Returns a `SkyToastInstance`." be here in this parameter or in the method description? Seems like I copied this over to the wrong spot and it should be moved up.

Reply:
You can remove that sentence altogether; it should infer the return type automatically.

Reply:
Done. Thanks!